tests/unit/changeset-test.js

```diff
@@ -1688,6 +1688,7 @@ module('Unit | Utility | changeset', function(hooks) {
 
     Object.defineProperty(myChangeset, 'isOptionSelected', {
       get() {
+        // eslint-disable-next-line ember/no-get
         return this.get('isOptionOne') || this.get('isOptionTwo') || this.get('isOptionThree');
```

Comment:
Can we ✂️ `this.get` and do `this.isOptionOne...`?

Reply:
No, it doesn't work without this.get

Reply:
Sorry can you explain why it doesn't work with `this[property]`?

Reply:
I think this is because at this point `this` is not a `Proxy`, it is a `BufferedChangeset`. So, if I don't explicitly write `this.get` it doesn't call BufferedChangeset.get.